File: Selenium/Activities/activity1.py
from selenium import webdriver
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.common.by import By

service = FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service = service) as driver:
  
  driver.get("https://v1.training-support.net")
  print("old tiltle is" ,driver.title)
  driver.find_element(By.ID, "about-link").click()
  print("New page title is: ", driver.title)
  driver.quit()

File: Selenium/Activities/Activity1.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.apache.commons.collections.bag.SynchronizedSortedBag;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity1 {

    public static void main(String[] args) {
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();

        // activity2
        driver.get("https://v1.training-support.net");

        System.out.println("Title is " +driver.getTitle());
        driver.findElement(By.id("about-link")).click();
        System.out.println("New Title is " +driver.getTitle());

        driver.quit();


    }
}
